src/platform/user/profile/vet360/components/base/TransactionErrorBanner.jsx

```diff
@@ -56,15 +43,8 @@ export function MVIError(props) {
     <AlertBox
       {...props}
       status="error"
-      content={
-        <div>
-          <h4>We can’t access your contact information right now</h4>
-          <p>
-            We’re sorry. Something went wrong on our end. Please refresh this
-            page or try again later.
-          </p>
-        </div>
-      }
+      headline=">We can’t access your contact information right now"
```

Comment:
There's a stray `>` in the headline. And can you add the extra bottom margin on this `AlertBox`?

Reply:
Done